build.gradle.kts

```diff
@@ -386,3 +386,13 @@ tasks.register<Zip>("allIncubationReportsZip") {
 }
 
 fun Project.collectAllIncubationReports() = subprojects.flatMap { it.tasks.withType(IncubatingApiReportTask::class) }
+
+// Ensure the archives produced are reproducible
+allprojects {
+    tasks.withType<AbstractArchiveTask>() {
```

Comment:
`.configureEach` is important. IIUC, without `configureEach` every task is realized.

Reply:
Forgot that, thanks, fixed now.